core/schema/project/asset_other_child.yml

```diff
@@ -5,7 +5,7 @@ type: "shotgun_entity"
 entity_type: "Asset"
 
 # the shotgun field to use for the folder name
-name: "{sg_asset_parent.Asset.sg_asset_type}/{sg_asset_parent.Asset.sg_asset_category}/{sg_asset_parent.Asset.code}/{sg_asset_category}/{code}"
+name: "{sg_asset_parent.Asset.sg_asset_type}/{sg_asset_parent.Asset.sg_asset_library}/{sg_asset_parent.Asset.sg_asset_category}/{sg_asset_parent.Asset.code}/{sg_asset_category}/{code}"
```

Comment:
Same question here, this order of folders (type/library/category) is different from the asset_child.yml (library/type/category). Do we want to unify this ordering?